lib/internal/Magento/Framework/Indexer/CacheContext.php

```diff
@@ -73,6 +73,6 @@ public function getIdentities()
                 $identities[] = $cacheTag . '_' . $id;
             }
         }
-        return array_merge($identities, array_unique($this->tags));
+        return array_unique(array_merge($identities, array_unique($this->tags));
```

Comment:
Missing a closing parenthesis, should be ))); 
Cause of most of the test failures